src/Microsoft.Framework.ApplicationHost2/Microsoft.Framework.ApplicationHost2.kproj

```diff
@@ -13,5 +13,8 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <ProduceOutputsOnBuild>True</ProduceOutputsOnBuild>
```

Comment:
Seriously? Are we going to just keep yanking this? Why isn't it being stored in the user file?